### PR TITLE
Implement QE feedback for 4.2

### PR DESCRIPTION
**Pull request by rh-max**
This PR implements Junqi's (QE) feedback for the 4.2 monitoring docs from #16586, #16752, #16474, and #16575. Only a few items were not implemented fully or at all (I commented on those items with explanations).
@juzhao Thank you for the review!

### CODE REVIEW

File: modules/monitoring-assigning-tolerations-to-monitoring-components.adoc [hunk 1]
```diff
@@ -30,14 +30,14 @@ metadata:
   namespace: openshift-monitoring
 data:
   config.yaml: |
-    *_component_*:
+    <component>:
       tolerations:
-        *_tolerations specification_*
+        <toleration specification>
```
Comment (jboxman) [inline modules/monitoring-assigning-tolerations-to-monitoring-components.adoc]: Per our style guide, we're replacing spaces with underscores: `<foo_variable>`

**Review by jboxman — COMMENTED** [review]
(My first use of the review button.)

One comment, but otherwise lgtm.

File: modules/monitoring-assigning-tolerations-to-monitoring-components.adoc (hunk 1 above)
Comment (rh-max) [reply modules/monitoring-assigning-tolerations-to-monitoring-components.adoc]: Implemented everywhere in monitoring docs.

**Comment by jboxman** [comment]
@rh-max, can you squash into a single commit and force-push? Then I'll merge. Thanks!

**Comment by rh-max** [comment]
@jboxman Done. Thank you!

**Comment by rh-max** [comment]
@jboxman I've made one other change:
https://github.com/openshift/openshift-docs/pull/16978/files#diff-cf9d582967542fd125f5df9012e5c2dbL42
It's all squashed now, so you can merge & cherry-pick. Thanks!

**Pull request merged**